src/resolve/resolved-dns-stub.c

```diff
@@ -564,6 +706,23 @@ int manager_dns_stub_start(Manager *m) {
         } else if (r < 0)
                 return log_error_errno(r, "Failed to listen on %s socket 127.0.0.53:53: %m", t);
 
+        if (!ordered_set_isempty(m->dns_extra_stub_listeners)) {
+                DNSStubListenerExtra *l;
+                Iterator i;
+
+                log_debug("Creating stub listener extra using %s.",
+                          m->dns_stub_listener_mode == DNS_STUB_LISTENER_UDP ? "UDP" :
+                          m->dns_stub_listener_mode == DNS_STUB_LISTENER_TCP ? "TCP" :
+                          "UDP/TCP");
+
+                ORDERED_SET_FOREACH(l, m->dns_extra_stub_listeners, i) {
+                        if (l->mode == DNS_STUB_LISTENER_UDP)
+                                r = manager_dns_stub_udp_fd_extra(m, l);
+                        else
+                                r = manager_dns_stub_tcp_fd_extra(m, l);
+                }
```

Comment:
{} not needed.

Reply:
```
                        if (IN_SET(l->mode, DNS_STUB_LISTENER_UDP, DNS_STUB_LISTENER_YES))
                                r = manager_dns_stub_udp_fd_extra(m, l);
                        if (IN_SET(l->mode, DNS_STUB_LISTENER_TCP, DNS_STUB_LISTENER_YES))
                                r = manager_dns_stub_tcp_fd_extra(m, l);
```
Moreover, it is better to make `DnsStubListenerMode` bitfield.

Reply:
As we are storing UDP and TCP in diffrent object, this is not required.